### ISSUE TITLE
Accrescent Store?

### ISSUE THREAD

**Issue by PrivacyAndSecurity** [gabeg805/NFC-Alarm-Clock#22]
Hey,

could you publish NFC Alarm Clock in the Accrescent Store? I would love to see you there and download the App from there.

**Comment by gabeg805** [comment]
Yeah I'll look into it!  I'm just one person, so I'm not sure when I'll be able to do it.  I've been trying to get my app on F-Droid, and if the process for Accrescent is similar, it's not as easy as just click and it's done.  There's a lot of steps behind the scenes that I have to do (for F-Droid) to get it done, and I still haven't even done that.

It's on my radar now though, so at some point, it'll get done!
